src/ep.config.json

```diff
@@ -2,7 +2,7 @@
   "cortexApi": {
     "path": "/cortex",
     "scope": "vestri",
-    "pathForProxy": "http://localhost:9080"
+    "pathForProxy": "http://vestri.us.epdemos.com"
```

Comment:
Can we revert this? OOTB we should be on `localhost:9080`

Reply:
yeah, good catch. I didn't mean to put this in. was leftover in my workspace